src/index.html

```diff
@@ -8,6 +8,59 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <header>Stars ww</header>
+
+    <main>
+      <div class="stars">
+        <div class="stars__star stars__star--stars--0"></div>
+        <div class="stars__star stars__star--stars--1"></div>
+        <div class="stars__star stars__star--stars--2"></div>
+        <div class="stars__star stars__star--stars--3"></div>
+        <div class="stars__star stars__star--stars--4"></div>
+        <div class="stars__star stars__star--stars--5"></div>
+      </div>
+      <div class="stars">
+        <div class="stars__star stars__star--stars--0"></div>
+        <div class="stars__star stars__star--stars--1"></div>
+        <div class="stars__star stars__star--stars--2"></div>
+        <div class="stars__star stars__star--stars--3"></div>
+        <div class="stars__star stars__star--stars--4"></div>
+        <div class="stars__star stars__star--stars--5"></div>
+      </div>
+      <div class="stars">
```

Comment:
```suggestion
      </div>
      
      <div class="stars">
```
Add empty lines between multiline sibling blocks of HTML:
![image](https://github.com/mate-academy/layout_stars/assets/63243136/fb07f701-1982-42e6-becd-ecbefc89f503)